src/cuter_scheduler_maxcover.erl

```diff
@@ -179,10 +191,22 @@ handle_info(_Msg, State) ->
                ; (request_operation, from(), state()) -> {reply, cuter_solver:solver_input() | try_later, state()}
                ; (stop, from(), state()) -> {stop, normal, ok, state()}
                ; (get_visited_tags, from(), state()) -> {reply, cuter_cerl:visited_tags(), state()}
-               ; (get_erroneous_inputs, from(), state()) -> {reply, cuter:erroneous_inputs(), state()}
+               ; (get_erroneous_inputs, from(), state()) -> {reply, [cuter:input()], state()}
+               ; (clear_erroneous_inputs, from(), state()) -> {reply, ok, state()}
                ; (get_solved_models | get_not_solved_models, from(), state()) -> {reply, non_neg_integer(), state()}
+               ; ({add_seed_input, cuter:input()}, from(), state()) -> {reply, ok, state()}
+               ; ({set_depth, pos_integer()}, from(), state()) -> {reply, ok, state()}
                .
 
+%% Add a seed input.
+handle_call({add_seed_input, SeedInput}, _From, #st{inputsQueue = InputsQueue}=S) ->
+  InpQueue = queue:in({1, SeedInput}, InputsQueue),
+  {reply, ok, S#st{ inputsQueue = InpQueue }};
```

Comment:
The extra spaces here and in the similar line below around `@st{` and `}` are inconsistent with the rest of the file.